src/ekka_mnesia.erl

```diff
@@ -315,14 +315,35 @@ connect(Node) ->
 
 %% @doc Running nodes.
 -spec(running_nodes() -> list(node())).
-running_nodes() -> cluster_nodes(running).
+running_nodes() ->
+    case ekka_rlog:role() of
+        core ->
+            CoreNodes = mnesia:system_info(running_db_nodes),
+            {Replicants0, _} = rpc:multicall(CoreNodes, ekka_rlog_status, replicants, [], 1000),
```

Comment:
handle timeout?

Reply:
hum.... maybe we shouldnt call with a timeout?

Reply:
Good catch, fixed